Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1067,6 +1068,13 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
     return pdfElementTypes;
 }
 
+#pragma mark Events
+
+static bool isContextMenuEvent(WebMouseEventButton button, OptionSet<WebEventModifier> modifiers)
+{
+    return button == WebMouseEventButton::Right || (button == WebMouseEventButton::Left && modifiers.contains(WebEventModifier::ControlKey));
```

Comment:
Do we really have to code this here, instead of getting it from NSEvent somehow? I don't know if "controlKey" is true for all keyboards/user configs

Reply:
Ah good point, we should probably use something like `isContextClick` from `WebFrame`

https://github.com/WebKit/WebKit/blob/bec01ed582b08a515921746208c15f0a641f6de7/Source/WebKit/WebProcess/WebPage/WebFrame.cpp#L1207-L1215

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/969e56be3a56363a7cd3acee00649dbe12ef1814